caikit_nlp/modules/text_generation/__init__.py

```diff
@@ -15,3 +15,4 @@
 # Local
 from .peft_prompt_tuning import PeftPromptTuning
 from .peft_tgis_remote import PeftPromptTuningTGIS
```

Comment:
Metacomment - in an unrelated PR, should we make the TGIS client? It's a bit awkward since a model does need to be bootstrapped, but I do feel like either this or the Peft prompt tuning block could be used to initialize it

Reply:
this is related to `.peft_tgis_remote` module right? Yeah, I think that make sense.